### ISSUE TITLE
ERROR: "HTTP Error 403: " when connecting to http://osrecovery.apple.com/InstallationPayload/RecoveryImage

### ISSUE THREAD

**Issue by leleliu008** [quickemu-project/quickemu#714]
#### Steps to reproduce the behaviour

run command `quickget macos monterey`

#### Quickemu output

```
macos-monterey/OpenCore.qcow2          100%[===========================================================================>]  22.06M   224KB/s    in 3m 1s   
macos-monterey/OVMF_CODE.fd            100%[===========================================================================>]   1.88M   114KB/s    in 19s     
macos-monterey/OVMF_VARS-1024x768.fd   100%[===========================================================================>] 128.00K  56.7KB/s    in 2.3s    
Downloading monterey...
ERROR: "HTTP Error 403: " when connecting to http://osrecovery.apple.com/InstallationPayload/RecoveryImage
Making macos-monterey.conf
Giving user execute permissions on macos-monterey.conf,

To start your macOS virtual machine run:
    quickemu --vm macos-monterey.conf
```

**sb_release --all**

```
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 23.04
Release:	23.04
Codename:	lunar
```

**uname -a**

```
Linux leleliu008 6.2.0-20-generic #20-Ubuntu SMP PREEMPT_DYNAMIC Thu Apr  6 07:48:48 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux
```

**quickemu --version**
4.7

**Comment by leleliu008** [comment]
fixed in version 4.8

**Comment by leleliu008** [comment]
It seems that VPN cann't be opened while executing  `quickget macos monterey`
